wp-content/plugins/core/src/Blocks/Types/Icon_Grid/Icon_Grid_Model.php

```diff
@@ -0,0 +1,44 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Blocks\Types\Icon_Grid;
+
+use Tribe\Project\Blocks\Types\Base_Model;
+use Tribe\Project\Templates\Components\blocks\icon_grid\Icon_Grid_Controller;
+use Tribe\Project\Templates\Components\link\Link_Controller;
+
+/**
+ * Class Icon_Grid_Model
+ *
+ * Responsible for mapping values from the block to arguments
+ * for the component
+ */
+class Icon_Grid_Model extends Base_Model {
+	public function get_data(): array {
+		return [
+			Icon_Grid_Controller::CLASSES     => $this->get_classes(),
```

Comment:
Just caught this in porting,  need to add: `Icon_Grid_Controller::ATTRS => $this->get_attrs(),` so we get any block ID / class data.